engine/installation/linux/docker-ce/debian.md

```diff
@@ -208,16 +208,15 @@ from the repository.
     ```bash
     $ apt-cache madison docker-ce
 
-    docker-ce | {{ minor-version }}.0~ce-0~debian-jessie | {{ download-url-base}} jessie/stable amd64 Packages
+    docker-ce | {{ minor-version }}.0~ce-0~debian | {{ download-url-base}} jessie/stable amd64 Packages
```

Comment:
These instruction changes look generic enough to be flexible for any package version format in the past and future. I'd prefer not to have to provide multiple names of the same identical package if possible.